framework/source/class/qx/bom/webfonts/Validator.js

```diff
@@ -34,16 +34,24 @@ qx.Class.define("qx.bom.webfonts.Validator", {
   /**
    * @param fontFamily {String} The name of the font to be verified
    * @param comparisonString {String?} String to be used to detect whether a font was loaded or not
+   * @param fontWeight {String} the weight of the font to be verified
+   * @param fontStyle {String} the style of the font to be verified
```

Comment:
since these two are optional too
```suggestion
   * @param fontWeight {String?} the weight of the font to be verified
   * @param fontStyle {String?} the style of the font to be verified
```